sdk/keyvault/keyvault-certificates/test/mergeAndImport.test.ts

```diff
@@ -75,6 +75,10 @@ describe("Certificates client - merge and import certificates", () => {
       undefined,
       "The signed certificate will never be the same, so we can't play it back."
     );
+    if (!isNode) {
+      // recorder.skip doesn't work on TEST_MODE=live
+      return this.skip();
```

Comment:
Why is this being skipped in browsers?

Reply:
We use openssl through Node's child_process to sign a certificate. Doing it on the browser is considerably harder.

Reply:
https://github.com/Azure/azure-sdk-for-js/pull/7444#discussion_r381518341
Here too.

Reply:
I'm not super familiar with the record + playback stuff. What's the difference between recorder.skip and this.skip? Reading the comment, I initially thought they were the same.

Reply:
`this` - mocha context object
`this.skip` is a mocha feature - basically skips the test
`recorder.skip` leverages `this.skip` internally -  can be used to skip the test in record and playback modes in node and browser selectively by providing arguments, recorder doesn't manage the live test mode.

Reply:
Thanks for that explanation @HarshaNalluru, very informative!